libs/sql-schema-describer/src/postgres.rs

```diff
@@ -205,16 +205,20 @@ impl SqlSchemaDescriber {
             };
             let tpe = get_column_type(data_type.as_ref(), &full_data_type, arity, enums);
 
+            //todo we might want to use regexes here
```

Comment:
yes we could use a regex, but part of it is variable, so they would have to be compiled for every field or (I think I already did that somewhere in the sql-schema-describer) capture the type, then match for the regex and the type capture, which is probably best for performance but also more complicated.

Reply:
And sql-schema-describer performance is already good enough™

Reply:
Ok, then we'll leave it as is for now.